code/src/components/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import TechnigoLogo from "../assets/logo.svg";
```

Comment:
Only components should be written in uppercase

Reply:
fixed :)